src/docs/sdk/event-payloads/properties/span_start_timestamp.mdx

```diff
@@ -0,0 +1,22 @@
+`start_timestamp`:
+
+: _Required_. A timestamp representing when the measuring started. The
+  format is either a string as defined in [RFC
+  3339](https://tools.ietf.org/html/rfc3339) or a numeric (integer or float)
+  value representing the number of seconds that have elapsed since the [Unix
+  epoch](https://en.wikipedia.org/wiki/Unix_time). Important to note is that the `start_timestamp`
+  must be before the `timestamp` otherwise we will discard the Span as invalid.
```

Comment:
```suggestion
  epoch](https://en.wikipedia.org/wiki/Unix_time). The `start_timestamp` value must be greater or
  equal the `timestamp` value, otherwise the Span is discarded as invalid.
```

It is valid to have start == end.